cou/utils/text_styler.py

```diff
@@ -37,3 +39,35 @@ def normal(text: str) -> str:
     :rtype: str
     """
     return Style.RESET_ALL + text + Style.RESET_ALL
+
+
+def prompt_message(parameter: str, default: Optional[str] = None) -> str:
```

Comment:
I think `default_choice` represents better this function argument. `default` is a little vague.

Reply:
heh, I do not like the `default_choice` is too long for no purpose. But I see your point, if we add `choices` or `options` input parameter (see my comment below) using only `default` will be even more intuitive.

Reply:
Since we are just using `Continue?` check, we should always have only `y` and `n` as options, at least in the print message. So I prefer keeping the options a constant in the function instead of making it a parameter

Reply:
That's why at the beginning I suggest to make `parameters` input argument instead, so we could use it without continue. However, right now we do not have use case for this, but I think it will come, and then we refactor all this functions again.